orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -19,37 +18,13 @@
 import torch
 from torch.utils.cpp_extension import ROCM_HOME
 
-ONNX_OPSET_VERSION = 12
-
-
-def _run_forward(execution_session, onnx_model, device, *inputs, **kwargs):
```

Comment:
promotoed to `@staticmethod GraphExecutionManager.execution_session_run_forward`